### ISSUE TITLE
when only part of the data set is trained, loss reaches 0

### ISSUE THREAD

**Issue by JiangOfCHINA** [DSPsleeporg/smiles-transformer#4]
Hello and thanks for publishing the code of your research. After the pre-training, I had some questions.

iter     0 | loss 3.823 | ppl 45.743
Train   1: iter  1000 | loss 0.003 | ppl 1.003
Train   1: iter  2000 | loss 0.001 | ppl 1.001

I want to do the pre-training by myself, but during the training, the loss value drops rapidly.I don't know if there is something wrong with training corpus(chembl_25. csv) or if it is normal.The training corpus(chembl_25. csv) refers to your prepare_data.ipynb script in expriments folder.

Part of  training corpus is shown in the following figure:
![image](https://user-images.githubusercontent.com/50139431/78033333-d4c31780-7398-11ea-8071-0fe849611485.png)
-------------------------------------------
![image](https://user-images.githubusercontent.com/50139431/78034289-199b7e00-739a-11ea-9fa4-d3496448684e.png)

![image](https://user-images.githubusercontent.com/50139431/78034374-3a63d380-739a-11ea-9e21-d8e971d9501d.png)
Is the file chembl_25. csv generated in the same way as the file chembl_24. csv?

**Comment by shionhonda** [comment]
It does reach the zero training loss when the model is large enough, but I don't really remember at which iteration it reaches. So my advice is to keep training for long enough and go ahead to the downstream tasks.

Yes, `chembl_25. csv` is generated in the same way as `chembl_24. csv`

**Comment by JiangOfCHINA** [comment]
thanks for your reply
